### ISSUE TITLE
Invalid HTML in Dataset Restore Monitor

### ISSUE THREAD

**Issue by paurkedal** [dCache/dcache#5071]
The "Status" column in `/poolInfo/restoreHandler/*` needs to be HTML-escaped or otherwise adjusted, since it sometimes contain the value `[<idle>]`.  Here is the full row from the document as we saw it (pnfs obscured):
```html
<tr>
<td class="pnfs">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</td>
<td class="subnet">0.0.0.0/0-Xrootd/2</td>
<td class="pool">N.N.</td>
<td class="started">09.17 16:54:44</td>
<td class="clients">1</td>
<td class="retries">0</td>
<td class="status">[<idle>]</td>
</tr>
```
We are running dcache-5.2.2-1 on Ubuntu 16.04.  In current master, it seems `modules/dcache/src/main/java/diskCacheV111/poolManager/HttpPoolMgrEngineV3.java` substitutes unescaped all fields, but most of them presumably have a restricted enough syntax.

**Comment by kofemann** [comment]
Thanks for reporting this!

**Comment by alrossi** [comment]
https://rb.dcache.org/r/11948/

**Comment by kofemann** [comment]
fixed in master and 5.2

**Comment by paurkedal** [comment]
thanks!
